src/bundle/ui-dev/src/modules/universal-discovery/bookmarks.tab.module.js

```diff
@@ -22,7 +22,11 @@ import { getIconPath } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scri
 import { getTranslator } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
 
 const BookmarksTabModule = () => {
-    const shouldRestorePreviousStateRef = useRef(true);
+    const didBookmarksMarkedLocationRef = useRef(false);
+    const restorationStateRef = useRef({
+        markedLocationId,
+        loadedLocationsMap,
```

Comment:
Wrong order, I think - you're using both `markedLocationId` and `loadedLocationsMap` before they're declared (in 33 and 37)

Reply:
Actually, I didn't want to set them to anything specific. I will just set the whole ref to null.